scrapy/pipelines/images.py

```diff
@@ -100,14 +101,20 @@ def file_downloaded(self, response, request, info):
     def image_downloaded(self, response, request, info):
         checksum = None
         for path, image, buf in self.get_images(response, request, info):
+            content_type = Image.MIME[image.format]
+            if self.convert_type != 'PRESERVE':
```

Comment:
I am not that well versed with the way the resolve method handles the value 'None', how does a typical user pass a value None? by passing a blank string?